.github/workflows/publish-stage.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Disable Indexing
         run: "sed -i 's/noIndex: false/noIndex: true/g' docusaurus.config.js"
       - name: Update URL
-        run: "sed -i 's!url: \"https://docs.camunda.io\"!url: \"https://stage.docs.camunda.io\"!g' docusaurus.config.js"
+        run: 'sed -i ''s!url: "https://docs.camunda.io"!url: "https://stage.docs.camunda.io"!g'' docusaurus.config.js'
```

Comment:
(no action required) 

This looks right to me, but just leaving a placeholder for myself to remember that this part of the staging deploy changed, in case it ends up being an issue.

Reply:
Good idea 👍 

TBH, this one was a surprise to me too, but apparently, YAML escapes quotes `'` with another quote `''`.